docs/source/executors/executors.rst

```diff
@@ -60,6 +68,20 @@ SSH Executor
         client_keys:
           - /opt/tardis/ssh/tardis
 
+    .. rubric:: Example configuration (Using Multi-factor Authentication)
+
+    .. code-block:: yaml
+
+      !TardisSSHExecutor
+        host: login.dorie.somewherein.de
+        username: clown
+        client_keys:
+          - /opt/tardis/ssh/tardis
+        mfa_secrets:
+          - prompt: "Enter 2FA Token:"
+            secret: "IMIZDDO2I45ZSTR6XDGFSPFDUY"
```

Comment:
Sorry, my head is a bit hung op on having `mfa_secrets`->`secret`. It duplicates the information without saying much.

- Instead of `mfa_secrets` I recommend to use `totp` so it's clear which method we are dealing with (not that I necessarily expect another, but it's free lunch).
- Instead of `secret` I could also see `key` working here, since that is what it's called often. However, `secret` has the added benefit of being just as long as `prompt` and I kind of like how the colons align... 🫣

Reply:
What about calling it `mfa_config` instead?
```
mfa_config:
  - prompt: "Enter 2 FA TOken:"
    secret: "IMIZDDO2I45ZSTR6XDGFSPFDUY"
```
Three colons aligned. :+1:

Reply:
However, I think 

```yaml
mfa_config:
  - prompt: "Enter 2 FA TOken:"
    totp: "IMIZDDO2I45ZSTR6XDGFSPFDUY"
```
is much better. Since it's clear that we are just supporting `totp`.